src/stories/components/CodeBlock.stories.mdx

```diff
@@ -21,10 +24,75 @@ import CodeBlock from '../../components/CodeBlock.vue';
         code: {
           default: text('Code', "// Render your payment form\n\nconst elements = stripe.elements();\n\nconst card = elements.create(1234);"),
         },
+        lineNumbers: {
+          default: boolean('Line Numbers', true),
+        },
+        editable: {
+          default: boolean('Editable Block', false),
+        },
+        background: {
+          default: select('Background Color', ['light', 'dark']),
+        },
+      },
+      template: `
+        <div class="p-16 flex justify-center w-1/2 m-auto" :class="{'bg-gray-500': background === 'dark'}">
+          <CodeBlock :language="language" :editable="editable" :line-numbers="lineNumbers">{{ code }}</CodeBlock>
```

Comment:
Changing this in the storybook doesn't update the preview

Reply:
I can only reproduce this on the code block group and I believe that's because of how the vnode is being rendered. You will need to switch tabs to see the updates. Would we ever be dynamically changing these values? I'm having a hard time finding a solution.

Reply:
Na I doubt it. Maybe just remove the bits that don't bind dynamically from the story